### ISSUE TITLE
Can not mark milestone as complete

### ISSUE THREAD

**Issue by vojtechsimetka** [Giveth/giveth-dapp#455]
Reported by @krrisis , I did not find out a reason why it does not work. He tried refresh, locking/unlocking wallet etc.

https://beta.giveth.io/campaigns/5b3789513a65c31e4e4e8328/milestones/5b71a86aa5f82545c38d2cc6

Description of the Error: 
  ```
Something went wrong with the transaction. Is your wallet unlocked? 
  Transaction link: 
  https://rinkeby.etherscan.io/tx/undefined => { 
"message": "gas required exceeds allowance or always failing transaction",
"code": -32603
}
```

![image](https://user-images.githubusercontent.com/7974813/44104346-e700b71a-9fee-11e8-8eff-fda4e1cb21df.png)

**Comment by perissology** [comment]
It was already markedAsComplete. But for some reason the event did show up in the events table.

The logs were getting super hard to read, so I spent some time configuring the logs to write to a rolling daily file in json format. This will make the files smaller so we can more easily isolate the time, as well as allow us to query & parse the log files using something like `jq`. Hopefully that will allow us to better debug these situations in the future.

In the meantime, I manually updated the db to reflect the correct state. This milestone now needs review
